tests/director-tests/Makefile

```diff
@@ -1 +1,51 @@
-release: ;
+APP_NAME = compass-director-tests
+APP_PATH = tests/director-tests
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190913-65b55d1
+SCRIPTS_DIR = $(realpath $(shell pwd)/../..)/scripts
+DIRECTOR_GRAPHQL_API = "http://compass-dev-director:3000"
+export DIRECTOR_GRAPHQL_API
+
+include $(SCRIPTS_DIR)/generic_make_go.mk
+
+.PHONY: generate-director-examples clean-up
+
+VERIFY_IGNORE := /vendor\|/automock\|/testdata
+
+verify:: generate-director-examples
+
+$(eval $(call test-local))
+
+# We have to override test-local, because we need to run director with database as docker containers and connected with custom network
+# and the container itsefl has to be connected to the network
+test-local: ;
```

Comment:
Can we run here the `run.sh` script?

Reply:
run.sh requires Director to be up and running, `test-local` can work in such case, but `make test` will always fail, because it will run it in the Docker container. IMO it is better to keep it as it is.